tests/acceptance/features/apiSharingNg/sharedWithMe.feature

```diff
@@ -1617,3 +1617,285 @@ Feature: an user gets the resources shared to them
         }
       }
       """
+
+
+  Scenario: user who is also a member of group lists the file shared with them when auto-sync is disabled
+    Given user "Carol" has been created with default attributes and without skeleton files
+    And group "grp1" has been created
+    And the following users have been added to the following groups
+      | username | groupname |
+      | Brian    | grp1      |
+      | Carol    | grp1      |
+    And user "Alice" has uploaded file with content "to share" to "textfile.txt"
+    And user "Brian" has disabled the auto-sync share
+    And user "Alice" has sent the following share invitation:
+      | resource        | textfile.txt |
+      | space           | Personal     |
+      | sharee          | Brian        |
+      | shareType       | user         |
+      | permissionsRole | Viewer       |
+    And user "Alice" has sent the following share invitation:
+      | resource        | textfile.txt |
+      | space           | Personal     |
+      | sharee          | grp1         |
+      | shareType       | group        |
+      | permissionsRole | Viewer       |
+    When user "Brian" lists the shares shared with him using the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": [
+          "value"
+        ],
+        "properties": {
+          "value": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "required": [
+                "@UI.Hidden",
+                "@client.synchronize",
+                "createdBy",
+                "eTag",
+                "file",
+                "id",
+                "lastModifiedDateTime",
+                "name",
+                "parentReference",
+                "remoteItem",
+                "size"
```

Comment:
Should all the properties in the required field be checked? If not maybe they can be skipped.

Reply:
imo all the mentioned properties should be checked here

Reply:
I didn't see these properties being checked, so mentioned it. So, we need to add checks for these properties as well.

Reply:
The complete response test is covered in previous tests for sync disabled state. IMO, we don't need to validate every item in this test coverage. The presence of items in response can be checked through `required`. When the auto sync is disabled the only change made in response is `"@client.synchronize"`